index.js

```diff
@@ -13,6 +13,12 @@ export function render ({ props, state }, setState) {
   const { items, onChange, removeOnHide } = props;
   const { activeTabIndex } = state;
 
+  if (process.browser) {
+    window.addEventListener('deku-tab-control:setState', function (e) {
+      setState({ activeTabIndex: e.detail.activeTab });
+    });
+  }
```

Comment:
@iefserge this coincides with https://github.com/micnews/desktop-site/pull/241
We needed a quick way to reset the sitenav tabs when you click on the header hamburger menu.